src/scene/sprite-tiling/TilingSprite.ts

```diff
@@ -246,19 +247,19 @@ export class TilingSprite extends Container implements View, Instruction
      * const sprite = new TilingSprite({texture: Texture.WHITE});
      * sprite.anchor.set(0.5); // This will set the origin to center. (0.5) is same as (0.5, 0.5).
      */
-    get anchor(): PointLike
+    get anchor(): ObservablePoint
     {
         return this._anchor;
     }
 
-    set anchor(value: PointData)
+    set anchor(value: PointData | number)
     {
-        this._anchor.x = value.x;
-        this._anchor.y = value.y;
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
should we disable this rule? // eslint-disable-next-line @typescript-eslint/no-unused-expressions